ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -1602,6 +1602,20 @@ literals:
 - valve_status:
   - OPEN
   - CLOSED
+
+- low_smoke_alarm: # for aspirating type smoke detectors
+  - ACTIVE
+  - INACTIVE
+- medium_smoke_alarm: # for aspirating type smoke detectors
```

Comment:
Please update this field to be `medium_smoke_intensity_alarm`

Reply:
noted. Thanks